src/dal/interface/interface.spec.ts

```diff
@@ -36,8 +36,8 @@ describe('MongoDb Database client interface', () => {
       uri = `${dbInitArgs.url}:${dbInitArgs.port}`;
       authInfo = {
         auth: {
-          user: 'root',
-          password: 'example'
+          user: process.env.MONGO_USERNAME || '',
+          password: process.env.MONGO_PASSWORD || ''
```

Comment:
I know this is probably for verbosity, but isn't this redundant with having the default values already by an empty string?

Reply:
Actually, I made this change because we don't want our test results to depend on the environment. Let's say, for example, you add an environment variable `MONGO_
USERNAME`. If you were to run the tests in that environment, and the expected `user` result was `''`, then the test would fail, since we're defaulting `user` to be an environment variable. Writing this test this way is designed to let us run our tests in all environments, regardless of currently set variables.